test/search_test.dart

```diff
@@ -220,15 +220,46 @@ void main() {
       });
 
       test('facetDistributions parameter', () async {
+        //setup
         var index = await createBooksIndex();
         var response = await index
             .updateSettings(IndexSettings(
-              filterableAttributes: ['tag'],
+              filterableAttributes: [ktag],
             ))
             .waitFor(client: client);
-        expect(response.status, 'succeeded');
+        //action
         var result = await index.search('prince', facets: ['*']);
+        //test
+        expect(response.status, 'succeeded');
```

Comment:
This test made sense, being placed close to the result of the response. It is part of the setup.

But in any case, it will help to find any possible issue when indexing, I see there is a point in doing that, but if the update settings fail, the search will return an imprecise value, breaking the test.

So, what do you think about removing those assertions?

Reply:
actually they are unnecessary since `waitFor` will only proceed after the task is completed